src/test/kotlin/com/tschuchort/compiletesting/KotlinCompilationTests.kt

```diff
@@ -623,6 +624,17 @@ class KotlinCompilationTests {
 		assertClassLoadable(result, "${KotlinTestProcessor.GENERATED_PACKAGE}.${KotlinTestProcessor.GENERATED_JAVA_CLASS_NAME}")
 	}
 
+	@Test
+	fun `checks configuration with pluginClasspaths`() {
+		val result = defaultCompilerConfig().apply {
+			sources = listOf(SourceFile.kotlin("kSource.kt", "class KSource"))
+			pluginClasspaths = listOf(File("./plugin.jar"))
+		}.compile()
+
+		assertThat(result.exitCode).isEqualTo(ExitCode.INTERNAL_ERROR)
```

Comment:
Why are you testing failure? Shouldn't we make sure that the compiler actually uses plugins that we give to it?

Reply:
I was thinking about it and I realized we'd need a plugin to know it. Do you have any idea? I didn't find any text about it in the `result.messages`.

Reply:
I mean, I tested that failure because I thought it was the only thing that I could test.

Reply:
We can leave this test in (it's always good to test the negative path as well!), but we definitely need a positive test that it works, too.